### ISSUE TITLE
FormData() with 'Content-Type': 'multipart/form-data'?

### ISSUE THREAD

**Issue by braco** [mzabriskie/axios#53]
Trying to send an image across the network. Can't figure out what isn't working.
Headers:
```
{ 'Content-Type': 'multipart/form-data' }
```
Content is a FormData Blob object:
```
https://developer.mozilla.org/en-US/docs/Web/Guide/Using_FormData_Objects
```
using the request API with
```
{url: '...', method: 'put', data: blob, headers: {}}
```

Data seems to get swallowed up and not sent by axios. Body reported by chrome as {}.

This was working with the 'rest' package before I switched over.

Any idea what might be going wrong?

**Comment by mzabriskie** [comment]
Sounds like a bug. I will have to investigate.

**Comment by mzabriskie** [comment]
This should be fixed with 0.5.4
